release/src/main/scripts/run_rc_validation.sh

```diff
@@ -100,6 +100,8 @@ HUB_ARTIFACTS_NAME=hub-linux-amd64-${HUB_VERSION}
ice" "beam-runners-jet" "beam-runners-local-java-core" "beam-runners-portability-java" "beam-runners-samza-job-server" "beam-runners-samza" "beam-runners-spark-3-job-server" "beam-runners-spark-3" "beam-runners-twister2" "beam-sdks-java-bom" "beam-sdks-java-core" "beam-sdks-java-expansion-service-app" "beam-sdks-java-expansion-service" "beam-sdks-java-extensions-arrow" "beam-sdks-java-extensions-avro" "beam-sdks-java-extensions-euphoria" "beam-sdks-java-extensions-google-cloud-platform-core" "beam-sdks-java-extensions-join-library" "beam-sdks-java-extensions-json-jackson" "beam-sdks-java-extensions-kryo" "beam-sdks-java-extensions-ml" "beam-sdks-java-extensions-protobuf" "beam-sdks-java-extensions-python" "beam-sdks-java-extensions-sbe" "beam-sdks-java-extensions-schemaio-expansion-service" "beam-sdks-java-extensions-sketching" "beam-sdks-java-extensions-sorter" "beam-sdks-java-extensions-sql-datacatalog" "beam-sdks-java-extensions-sql-expansion-service" "beam-sdks-java-extensions-sql-hcatalog" "beam-sdks-java-extensions-sql-jdbc" "beam-sdks-java-extensions-sql-udf" "beam-sdks-java-extensions-sql-zetasql" "beam-sdks-java-extensions-sql" "beam-sdks-java-extensions-timeseries" "beam-sdks-java-extensions-zetasketch" "beam-sdks-java-fn-execution" "beam-sdks-java-google-cloud-platform-bom" "beam-sdks-java-harness" "beam-sdks-java-io-amazon-web-services" "beam-sdks-java-io-amazon-web-services2" "beam-sdks-java-io-amqp" "beam-sdks-java-io-azure" "beam-sdks-java-io-cassandra" "beam-sdks-java-io-cdap" "beam-sdks-java-io-clickhouse" "beam-sdks-java-io-common" "beam-sdks-java-io-contextualtextio" "beam-sdks-java-io-csv" "beam-sdks-java-io-debezium-expansion-service" "beam-sdks-java-io-debezium" "beam-sdks-java-io-elasticsearch" "beam-sdks-java-io-expansion-service" "beam-sdks-java-io-file-schema-transform" "beam-sdks-java-io-google-cloud-platform-expansion-service" "beam-sdks-java-io-google-cloud-platform" "beam-sdks-java-io-hadoop-common" "beam-sdks-java-io-hadoop-file-system" "beam-sdks-java-io-hadoop-format" "beam-sdks-java-io-hbase" "beam-sdks-java-io-hcatalog" "beam-sdks-java-io-influxdb" "beam-sdks-java-io-jdbc" "beam-sdks-java-io-jms" "beam-sdks-java-io-kafka" "beam-sdks-java-io-kinesis-expansion-service" "beam-sdks-java-io-kinesis" "beam-sdks-java-io-kudu" "beam-sdks-java-io-mongodb" "beam-sdks-java-io-mqtt" "beam-sdks-java-io-neo4j" "beam-sdks-java-io-parquet" "beam-sdks-java-io-pulsar" "beam-sdks-java-io-rabbitmq" "beam-sdks-java-io-redis" "beam-sdks-java-io-singlestore" "beam-sdks-java-io-snowflake-expansion-service" "beam-sdks-java-io-snowflake" "beam-sdks-java-io-solr" "beam-sdks-java-io-sparkreceiver-2" "beam-sdks-java-io-splunk" "beam-sdks-java-io-synthetic" "beam-sdks-java-io-thrift" "beam-sdks-java-io-tika" "beam-sdks-java-io-xml" "beam-sdks-java-maven-archetypes-examples" "beam-sdks-java-maven-archetypes-gcp-bom-examples" "beam-sdks-java-maven-archetypes-starter" "beam-sdks-java-nexmark" "beam-sdks-java-test-utils" "beam-sdks-java-tpcds" )
```

Comment:
Could we pull out the list of artifacts into a config file?

Reply:
Done